setup.py

```diff
@@ -0,0 +1,65 @@
+"""
+Code to allow this package to be pip-installed
+"""
+
+import os
+import sys
+
+from setuptools import setup
+
+LIBRARY_VERSION = '0.0.5'
+
+CURRENT_PYTHON = sys.version_info[:2]
+REQUIRED_PYTHON = (3, 6)
+
+if CURRENT_PYTHON < REQUIRED_PYTHON:
+    sys.stderr.write('''
+==========================
+Unsupported Python version
+==========================
+This version of esp-sdk requires Python {}.{}, but you're trying to
+install it on Python {}.{}.
+'''.format(*(REQUIRED_PYTHON + CURRENT_PYTHON)))
+    sys.exit(1)
+
+CUR_DIRECTORY_PATH = os.path.abspath(os.path.dirname(__file__))
+LIB_NAME = 'covid-xprize'
+
+
+def read(fname):
+    """
+    Read file contents into a string
+    :param fname: File to be read
+    :return: String containing contents of file
+    """
+    with open(os.path.join(os.path.dirname(__file__), fname)) as file:
+        return file.read()
+
+
+# Note: we use a direct URL for leaf-common rather than dependency-links, because the pip team has deprecated
+# the latter. See: https://github.com/pypa/pip/issues/4187#issuecomment-415067034
+setup(
+    name=LIB_NAME,
+    version=LIBRARY_VERSION,
+    python_requires='>={}.{}'.format(*REQUIRED_PYTHON),
+    packages=[
+        'covid_xprize/examples',
+        'covid_xprize/validation'
+    ],
+    install_requires=[
+        'keras==2.4.3',
```

Comment:
This is ultra strict. We may want to open these up a little and not insist on exact point versions. But probably okay for now.